packages/docs-reanimated/docs/core/useSharedValue.mdx

```diff
@@ -67,6 +67,8 @@ import SharedValueSrc from '!!raw-loader!@site/src/examples/SharedValue';
 
 - When you change the `sv.value` Reanimated will update the styles and keep the shared value in sync between the threads. However, this won't trigger a typical React re-render because a shared value is a plain JavaScript object.
 
+- When you read the `sv.value` on the [JavaScript thread](/docs/fundamentals/glossary#javascript-thread), the thread will get blocked until the value is fetched from the [UI thread](/docs/fundamentals/glossary#ui-thread). In most cases it will be negligible, but if you are running a lot of animations or reading a value multiple times, the wait time needed to synchronize both threads may significantly increase.
```

Comment:
"but if you are running a lot of animations" - This suggests that the animations themselves can impact JS thread performance. I think a more accurate statement would be something more like this - "but if the UI thread is busy" 🤔 what do you think?

Reply:
Sounds good to me